packages/components/src/components/brandshape/brandshape.ts

```diff
@@ -206,6 +260,82 @@ export default class SdBrandshape extends SolidElement {
       :host {
         @apply block;
       }
+
+      .container--stylized {
+        background-color: transparent;
+        padding: 40px 0;
+        position: relative;
+        z-index: 1;
+        border-width: 0;
+      }
```

Comment:
Would recommend to add this as class in HTML: `relative py-10 px-0 bg-transparent border-0`. 

@coraliefeil @mariohamann Do we need a new token for z-index: 1 or do we want to change it to z-10?

Reply:
okay, saw that you use it in 2 different lines. so please use it as tailwind and add @apply